### PR TITLE
Lossless 587: Changes according to the advisor review

### PR DESCRIPTION
**Pull request by Ignacio-Freire**

### CODE REVIEW

File: contracts/LosslessControllerV2.sol [hunk 1]
```diff
@@ -114,6 +114,7 @@ contract LosslessControllerV2 is Initializable, ContextUpgradeable, PausableUpgr
     // @notice Set a guardian contract.
     // @dev Guardian contract must be trusted as it has some access rights and can modify controller's state.
     function setGuardian(address newGuardian) external onlyLosslessAdmin whenNotPaused {
+        require(newGuardian != address(0), "LSS: Cannot be zero address");
```
Comment (DomantasP) [inline contracts/LosslessControllerV2.sol:117]: please add a new test case for this require

File: contracts/LosslessControllerV1.sol [hunk 2]
```diff
@@ -26,42 +26,48 @@ contract LosslessControllerV1 is Initializable, ContextUpgradeable, PausableUpgr
         _;
```
Comment (DomantasP) [inline contracts/LosslessControllerV1.sol:26]: Maybe let's not modify this contract cause it's already deployed.

**Review by DomantasP — CHANGES_REQUESTED** [review]

File: contracts/LosslessControllerV2.sol (hunk 1 above)
Comment (Ignacio-Freire) [reply contracts/LosslessControllerV2.sol:117]: Added!

File: contracts/LosslessControllerV1.sol (hunk 2 above)
Comment (Ignacio-Freire) [reply contracts/LosslessControllerV1.sol]: Reverted!

**Review by DomantasP — APPROVED** [review]

**Pull request merged**